lib/feature/feature.go

```diff
@@ -11,14 +11,16 @@ const (
 	FlagEVMStakingModule Flag = "evm-staking-module"
 )
 
+var enabledFlags = make(map[Flag]bool)
```

Comment:
this will maybe fail when run with -race. Well need to wrap it in an atomic or use a lock possibly

Reply:
also maybe add a comment why we use globals here. Saying something about cometbft that doesn't expose the its root context, and the cosmos doesn't actually use the context provided in abci, so using context inside modules isn't possible.

Reply:
SG!